enrollment-server-common/src/main/java/com/wultra/app/enrollmentserver/common/onboarding/OnboardingProcessException.java

```diff
@@ -15,7 +15,7 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package com.wultra.app.onboardingserver.errorhandling;
+package com.wultra.app.enrollmentserver.common.onboarding;
```

Comment:
Can we keep this class in `errorhandling` package?

Reply:
Yes, of course, I have been hesitating about that. I will move it to `com.wultra.app.enrollmentserver.common.onboarding.errorhandling`